bin/reth/src/dirs.rs

```diff
@@ -338,17 +301,53 @@ impl<D> From<ChainPath<D>> for PathBuf {
     }
 }
 
+/// A type representing a path to the reth net directory.
+#[derive(Clone, Debug, PartialEq)]
+pub struct NetPath<D>(PlatformPath<D>);
+
+impl<D> NetPath<D> {
+    /// Returns the path to the default reth known peers file for this net directory.
+    pub fn known_peers_path(&self) -> PathBuf {
+        self.0.join("known_peers.json").into()
+    }
+}
+
+/// A type representing a path to the reth jwtsecret directory.
+#[derive(Clone, Debug, PartialEq)]
+pub struct JwtSecretPath<D>(PlatformPath<D>);
+
+impl<D> JwtSecretPath<D> {
+    /// Returns the path to the default reth jwtsecret file for this jwtsecret directory.
+    pub fn jwtsecret_path(&self) -> PathBuf {
+        self.0.join("jwt.hex").into()
+    }
+}
```

Comment:
same - keep the `jwt.hex` at top level dir, let's not add all these directories, it's too much

Reply:
done